### PR TITLE
Field Settings Refresh: Modularize SCSS Semantically

### PR DESCRIPTION
**Pull request by shervElmi**
This PR focuses on restructuring the existing `frm_admin.css` into semantically organized SCSS modules. **It does not introduce any SCSS-specific features yet**, this step is purely about improving structure and maintainability.

Once this PR is reviewed and approved, a follow-up PR will handle applying SCSS features and further optimization.

### How to test
- Run `npm run watch` to generate the new `frm_admin.css` and `font_icons.css`.
- Compare them with the versions from the "master" branch.
- Make sure the styles look and behave the same.

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@wordpress/stylelint-config/scss",
+  "rules": {
+	"no-descending-specificity": null,
+	"selector-class-pattern": null,
+	"selector-id-pattern": null,
+	"font-family-name-quotes": null,
+	"function-url-quotes": null,
+	"value-keyword-case": ["lower", {
+      "camelCaseSvgKeywords": true,
```
Comment (Crabcyborg) [inline .stylelintrc.json:10]: The spacing here seems inconsistent. It looks like a mix of tabs and spaces.

Comment (shervElmi) [reply .stylelintrc.json]: Addressed with the [Format stylelint config with proper indentation](https://github.com/Strategy11/formidable-forms/pull/2311/commits/1452f604dec447a15ca35645b6503c69506f639f) commit.

File: css/font_icons.css [hunk 2]
```diff
@@ -1,654 +1 @@
-@font-face {
```
Comment (Crabcyborg) [inline css/font_icons.css:1]: @shervElmi This file is actually loaded directly in some cases. Like in `FrmElementorController::register_elementor_hooks`.

Can we just keep this file the way it was?

File: css/frm_admin.css [hunk 3]
```diff
@@ -1,10150 +1,3 @@
-@import url(font_icons.css);
```
Comment (Crabcyborg) [inline css/frm_admin.css:1]: @shervElmi We may load this file directly as well in some cases?

Can we just have the CSS output create this file at this same path? That would be ideal.

**Review by Crabcyborg — APPROVED** [review]
Hm, I got a bit mixed up looking at the diffs and deleted some comments.

I don't see any issues.

I'll try getting this on QA for some extra testing to make sure none of the styles are messed up anywhere.

This looks great though.

Thanks @shervElmi!

🚀

**Comment by shervElmi** [comment]
Thanks @Crabcyborg, I’ll work on SCSS files more deeply as part of my rock task (Field Settings Refresh) and aim to improve them through that. 🚀 ❤️
